docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+#### path_gc_check_step
+
+- **Default:** 1,000 (Maximum Number of Files Scanned Continuously Each Time)
+- **Description:** The maximum number of files that can be scanned continuously each time.
+
+#### path_gc_check_step_interval_ms
+
+- **Default:** 10 ms
+- **Description:** The time interval between file scans.
+
+#### path_scan_interval_second
+
+- **Default:** 86,400 seconds
+- **Description:** The time interval at which GC cleans expired data.
+
+#### storage_flood_stage_usage_percent
+
+- **Default:** 95% (Storage Usage Threshold for Load and Restore Job Rejection)
+- **Description:** If the storage usage (in percentage) of the BE storage directory exceeds this value and the remaining storage space is less than `storage_flood_stage_left_capacity_bytes`, Load and Restore jobs are rejected.
+
+#### storage_flood_stage_left_capacity_bytes
+
+- **Default:** 107,374,182,400 Bytes (Remaining Storage Space Threshold for Load and Restore Job Rejection)
+- **Description:** If the remaining storage space of the BE storage directory is less than this value and the storage usage (in percentage) exceeds `storage_flood_stage_usage_percent`, Load and Restore jobs are rejected.
+
+#### tablet_meta_checkpoint_min_new_rowsets_num
+
+- **Default:** 10 (Minimum Number of Rowsets to Trigger a TabletMeta Checkpoint)
+- **Description:** The minimum number of rowsets to create since the last TabletMeta Checkpoint.
+
+#### tablet_meta_checkpoint_min_interval_secs
+
+- **Default:** 600 seconds
+- **Description:** The time interval of thread polling for a TabletMeta Checkpoint.
+
+#### max_runnings_transactions_per_txn_map
+
+- **Default:** 100 (Maximum Number of Concurrent Transactions per Partition)
+- **Description:** The maximum number of transactions that can run concurrently in each partition.
+
+#### tablet_max_pending_versions
+
+- **Default:** 1,000 (Maximum Number of Pending Versions on a Primary Key Tablet)
+- **Description:** The maximum number of pending versions that are tolerable on a Primary Key tablet. Pending versions refer to versions that are committed but not applied yet.
+
+#### tablet_max_versions
+
+- **Default:** 1,000 (Maximum Number of Versions Allowed on a Tablet)
+- **Description:** The maximum number of versions allowed on a tablet. If the number of versions exceeds this value, new write requests will fail.
+
+#### max_hdfs_file_handle
+
+- **Default:** 1,000 (Maximum Number of HDFS File Descriptors)
+- **Description:** The maximum number of HDFS file descriptors that can be opened.
+
+#### be_exit_after_disk_write_hang_second
+
+- **Default:** 60 seconds
+- **Description:** The length of time that the BE waits to exit after the disk hangs.
+
+#### min_cumulative_compaction_failure_interval_sec
+
+- **Default:** 30 seconds
+- **Description:** The minimum time interval at which Cumulative Compaction retries upon failures.
+
+#### size_tiered_level_num
+
+- **Default:** 7 (Number of Levels for Size-tiered Compaction)
```

Comment:
Can you please check all "Default" starting from here till the end of this doc to see if they contain a parameter description wrapped in parentheses, and remove such descriptions as they are unnecessary?